src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface State {
+  pressedKey: string | null;
+}
+
+export class App extends React.Component {
```

Comment:
Since you are using TypeScript, it's a good practice to define the Props type for the component. You can define an empty interface if your component doesn't have any props. For example, `interface Props {}` and then use it like `class App extends React.Component<Props, State>`.